src/makeCalculator.js

```diff
@@ -38,6 +38,29 @@
  */
 function makeCalculator() {
   // write code here
+
+  const calculator = {
+
+    result: 0,
+
+    add: (result, b) => result + b,
+    multiply: (result, b) => result * b,
+    divide: (result, b) => result / b,
+    subtract: (result, b) => result - b,
```

Comment:
This functions maybe don't know about some result
Will be better to name arguments as a, b (just some two values for calculations)